be/src/exprs/agg/distinct.h

```diff
@@ -104,38 +108,37 @@ struct DistinctAggregateState<LT, SumLT, FixedLengthLTGuard<LT>> {
         return sum;
     }
 
-    HashSet<T> set;
+    int64_t memory_usage = 0;
```

Comment:
prefer add a wrap function for memory_usage.
```
int64_t mem_usage() const {
 return _memory_usage;
}
```

Reply:
> prefer add a wrap function for memory_usage.
> 
> ```
> int64_t mem_usage() const {
>  return _memory_usage;
> }
> ```

emm……but no one will use it for now. so I think it's not necessary

Reply:
if direct using memory_usage, the following code make me confuse. I don't know who update the memory_usage
```
        phmap::InMemoryInput input(reinterpret_cast<const char*>(src));
        int64_t prev_memory = memory_usage;
        auto old_size = set.size();
        if (old_size == 0) {
            set.load(input);
        } else {
            int64_t temp = 0;
            HashSetWithMemoryCounting<T> set_src{CountingAllocator<T>(&temp)};
            set_src.load(input);
            set.merge(set_src);
        }
        return memory_usage - prev_memory;
```

Reply:
fixed